Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```diff
@@ -283,6 +283,37 @@
   components:
   - type: Unremoveable
 
+- type: entity
+  parent: ClothingHandsBase
+  id: ClothingHandsGlovesNorthStar
+  name: gloves of the north star
+  description: These gloves allow you to punch incredibly fast
+  components:
+  - type: Sprite
+    sprite: Clothing/Hands/Gloves/northstar.rsi
+  - type: Clothing
+    sprite: Clothing/Hands/Gloves/northstar.rsi
+  - type: MeleeWeapon
+    attackRate: 4
+    damage:
+      types:
+       Blunt: 7.5
```

Comment:
Think I should reduce this down to 7 or 6? 30 DPS might be a bit high, and I'd rather it be underpowered at first than overpowered.

Reply:
Honestly try bumping it up to 8 or 9

This is against unarmored targets:
4 hits with an esword (that can reflect projectiles) - 122 damage
14 hits with GOTNS - 107 damage